release-notes/6.0/preview/api-diff/rc1/WindowsDesktop/6.0.0-rc1_System.Windows.Forms.md

```diff
@@ -0,0 +1,19 @@
+# System.Windows.Forms
+
+``` diff
+ namespace System.Windows.Forms {
+     public class Control : Component, IArrangedElement, IBindableComponent, IComponent, IDisposable, IDropTarget, IHandle, IKeyboardToolTip, Interop.Ole32.IOleControl, Interop.Ole32.IOleInPlaceActiveObject, Interop.Ole32.IOleInPlaceObject, Interop.Ole32.IOleObject, Interop.Ole32.IOleWindow, Interop.Ole32.IPersist, Interop.Ole32.IPersistStorage, Interop.Ole32.IPersistStreamInit, Interop.Ole32.IQuickActivate, Interop.Ole32.IViewObject, Interop.Ole32.IViewObject2, Interop.Oleaut32.IPersistPropertyBag, ISupportOleDropSource, ISynchronizeInvoke, IWin32Window {
++        public bool IsAncestorSiteInDesignMode { get; }
+     }
+     public class Form : ContainerControl {
++        public bool MdiChildrenMinimizedAnchorBottom { get; set; }
+     }
+     public static class ProfessionalColors {
++        public static Color StatusStripBorder { get; }
+     }
+     public class ProfessionalColorTable {
++        public virtual Color StatusStripBorder { get; }
+     }
+ }
+```
+
```

Comment:
lgtm, let's have @RussKie confirm.

Reply:
@KlausLoeffelmann , I see few VB specific APIs missing here. Can you please confirm? https://github.com/dotnet/core/pull/6712

Reply:
@dreddy-work: Which do you have in mind?

Reply:
Properties in `ApplyApplicationDefaultsEventArgs ` class.

Reply:
Yes. We breaking-changed them (omitted) intentionally. They are replaced by the `Default`-styled properties we have now. Good catch, thanks for double-checking!!

Reply:
I cross checked and they are there. It was a confusion.